public/components/marketing-cards/28.html

```diff
@@ -0,0 +1,30 @@
+<style>
+    .background-animate {
+      background-size: 400%;
+      -webkit-animation: AnimationName 1.5s ease infinite;
+      animation: AnimationName 1.5s ease infinite;
+    }
+    @keyframes AnimationName {
+      0%,
+      100% {
+        background-position: 0% 50%;
+      }
+      50% {
+        background-position: 100% 50%;
+      }
+    }
+    .bg-image {
+      background-image: url("https://images.pexels.com/photos/10027233/pexels-photo-10027233.jpeg?auto=compress&cs=tinysrgb&w=1600&lazy=load");
+    }
+    .bg-image:hover {
+      background-image: url("https://images.pexels.com/photos/1252869/pexels-photo-1252869.jpeg?auto=compress&cs=tinysrgb&w=1600");
+    }
+  </style>
```

Comment:
Can't have this amount of custom CSS, it's a Tailwind CSS component library.

Also, the name `AnimationName` is not descriptive enough.

Reply:
whether can I move these keyframes to `tailwind.config.js`